html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -284,6 +439,39 @@ export default {
     }
   },
   methods: {
+    ifTab (set) {
+      return this.$refs.tabs && set.includes(this.$refs.tabs.tabs[this.tabIndex].title)
+    },
+    applyReevaluateAccess () {
+      this.$store.dispatch(`${this.$options.storeName}/reevaluateAccessNode`, this.mac).then(response => {
+        // noop
+      }).catch(() => {
+        // noop
+      })
+    },
+    applyRefreshFingerbank () {
+      this.$store.dispatch(`${this.$options.storeName}/refreshFingerbankNode`, this.mac).then(response => {
+        // noop
+      }).catch(() => {
+        // noop
+      })
+    },
+    applyRestartSwitchport () {
+      this.$store.dispatch(`${this.$options.storeName}/restartSwitchportNode`, this.mac).then(response => {
+        // noop
+      }).catch(() => {
+        // noop
+      })
+    },
```

Comment:
Remove both `.then` and `.catch` calls.

Reply:
I'll expand these both when I have a better idea how a successful API response looks.